new-dti-website/src/app/products/page.tsx

```diff
@@ -1,10 +1,177 @@
+              <span className="text-white font-medium"> </span>
+              <span className="text-neutral-200 italic">impact</span>
+            </div>
+          </div>
+          <p className="text-[#FFFFFF] w-[475px] md:text-lg md:leading-[21.78px] text-[14px] leading-[16.94px]  md:max-w-md max-w-[314px]">
+            Each of our projects address an unfulfilled need that exists in our community using
+            <span className="font-semibold"> human-centered design and software engineering.</span>
+          </p>
+        </div>
+      </div>
+      <ImageCarousel items={productIcons} />
+      <div className="w-full py-12" />
+
+      {products.current.map((product, index) => (
+        <div key={product.alt}>
+          <ProductDisplay orientation={index % 2 === 0 ? 'left' : 'right'} product={product} />
+          {index < products.current.length - 1 && (
+            <Connector
+              orientation={index % 2 === 0 ? 'right' : 'left'}
+              width={780}
+              height={470}
+              strokeWidth={6}
+            />
+          )}
+        </div>
+      ))}
+      <Connector
+        orientation={'left'}
+        width={780}
+        height={470}
+        strokeWidth={6}
+        text="Coming Soon..."
+      />
+      {products.upcoming.map((product, index) => (
+        <div key={product.alt}>
+          <ProductDisplay orientation={index % 2 === 0 ? 'left' : 'right'} product={product} />
+          {index < products.upcoming.length - 1 && (
+            <Connector
+              orientation={index % 2 === 0 ? 'right' : 'left'}
+              width={780}
+              height={470}
+              strokeWidth={6}
+            />
+          )}
+        </div>
+      ))}
+      <Connector
+        orientation="right"
+        width={390}
+        height={466}
+        strokeWidth={6}
+        className="!w-fit !ml-[50%]"
+      />
+      <div className="flex relative justify-center lg:my-32 md:my-64 mb-60 mt-40">
+        <RedBlob
+          className={'-left-52 bottom-0 scale-50 sm:scale-75 md:scale-100'}
+          intensity={0.3}
+        />
+        <RedBlob
+          className={'-right-52 bottom-0 scale-50 sm:scale-75 md:scale-100'}
+          intensity={0.3}
+        />
+        <div className="flex flex-col text-white max-w-screen-md text-center items-center space-y-6">
+          <p className="font-semibold text-[32px]">Have Any Ideas?</p>
+          <p className="px-20">
+            We've learned that tackling the hardest problems is the only way to truly create value
+            for the people around us.
+          </p>
+          <Button
+            className="text-white font-bold bg-[#D63D3D] hover:bg-[#A52424] hover:text-white border-none px-4 py-5"
+            variant="outline"
+            size="default"
+          >
+            Contact us
+          </Button>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+const ProductDisplay = (props: {
```

Comment:
Products seem to be associated with the wrong image due to spacing, e.g. the qmi images are closer to the cp text than is the cp images.
![image](https://github.com/user-attachments/assets/e9341cc9-4ed4-4a9a-a729-e30f3f6c09d0)